docs/manual/manual_throttle.rst

```diff
@@ -0,0 +1,346 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _manual_throttle:
+
+***********************************************
+Throttle Commands and Modes
+***********************************************
+Vertiq modules can receive throttle commands from a variety of sources, and interpret them differently based on the module's configurations. The following section
+provides greater detail on what exactly a throttle command is and how Vertiq modules interpret them.
+
+Note that generally a module needs to be :ref:`armed <manual_advanced_arming>` in order to spin when it receives a throttle command. Refer to the :ref:`manual_advanced_arming` section 
+for more information on arming.
+
+Module Support
+===============
+The throttle parsing and modes described here apply to speed modules. While servo modules can also receive commands to spin, they are not considered 
+throttle commands for the purposes of this documentation. Controlling servos will be covered in a separate section.
+
+Speed Modules
+**************
+
+.. table:: Speed Module Support for Throttle Commands and Modes
+
+	+-------------+------------------------------------+
+	| Module      | Throttle Support                   |
+	+-------------+------------------------------------+
+	| Vertiq 8108 | .. centered:: |:white_check_mark:| |
+	+-------------+------------------------------------+
+	| Vertiq 4006 | .. centered:: |:white_check_mark:| |
+	+-------------+------------------------------------+
+	| Vertiq 2306 | .. centered:: |:white_check_mark:| |
+	+-------------+------------------------------------+
+
+Servo Modules
+**************
+Servo modules do not support the throttle parsing and modes described in this section of the documentation, documentation for servo module parsing will be provided in a

```

Comment:
I don't know that this is a necessary sentence. The big scary Xs get the point across that throttle isn't supported. In the module definition/intro file we can say "hey if you're using servo then you should probably read up on the servo parser"